src/_includes/docs/add-to-app/ios-project/link-and-embed.md

```diff
@@ -0,0 +1,155 @@
+you can't publish that app to the App Store.
+Publishing fails with a
+`Found an unexpected Mach-O header code` archive error.
+
+##### Link all frameworks
+
+To link the necessary frameworks, follow this procedure.
+
+1. Choose the frameworks to link.
+
+   1. In the **Project Navigator**, click on your project.
+
+   1. Click the **Build Phases** tab.
+
+   1. Expand **Link Binary With Libraries**.
+
+      {% render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/linked-libraries.png",
+      caption:"Expand the **Link Binary With Libraries** build phase in Xcode" %}
+
+   1. Click **+** (plus sign).
+
+   1. Click **Add Other...** then **Add Files...**.
+
+   1. From the **Choose frameworks and libraries to add:** dialog box,
+      navigate to the `/path/to/MyApp/Flutter/Release/` directory.
+
+   1. Command-click the frameworks in that directory then click **Open**.
+
+      {% render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/choose-libraries.png",
+      caption:"Choose frameworks to link from the **Choose frameworks and
+      libraries to add:** dialog box in Xcode" %}
+
+1. Update the paths to the libraries to account for build modes.
+
+   1. Launch the Finder.
+
+   1. Navigate to the `/path/to/MyApp/` directory.
+
+   1. Right-click on `MyApp.xcodeproj` and select **Show Package
+      Contents**.
+
+   1. Open `project.pbxproj` with Xcode. The file opens in Xcode's text
+      editor. This also locks **Project Navigator** until you close the text editor.
+
+      {% render docs/captioned-image.liquid,
+      image:"development/add-to-app/ios/project-setup/project-pbxproj.png",
+      caption:"The `project-pbxproj` file open in the Xcode text editor" %}
+
+   1. Find the lines that resemble the following text in the
+      `/* Begin PBXFileReference section */`.
+
+      ```text
+      312885572C1A441C009F74FF /* Flutter.xcframework */ = {
+        isa = PBXFileReference;
+        expectedSignature = "AppleDeveloperProgram:S8QB4VV633:FLUTTER.IO LLC";
+        lastKnownFileType = wrapper.xcframework;
+        name = Flutter.xcframework;
+        path = Flutter/[!Release!]/Flutter.xcframework;
+        sourceTree = "<group>";
+      };
+      312885582C1A441C009F74FF /* App.xcframework */ = {
+        isa = PBXFileReference;
+        lastKnownFileType = wrapper.xcframework;
+        name = App.xcframework;
+        path = Flutter/[!Release!]/App.xcframework;
+        sourceTree = "<group>";
+      };
+      ```
+
+   1. Change the `Release` text highlighted in the prior step
+      and change it to `$(CONFIGURATION)`.
+
+      ```text
+      312885572C1A441C009F74FF /* Flutter.xcframework */ = {
+        isa = PBXFileReference;
+        expectedSignature = "AppleDeveloperProgram:S8QB4VV633:FLUTTER.IO LLC";
+        lastKnownFileType = wrapper.xcframework;
+        name = Flutter.xcframework;
+        path = Flutter/[!$(CONFIGURATION)!]/Flutter.xcframework;
```

Comment:
```suggestion
        path = "Flutter/[!$(CONFIGURATION)!]/Flutter.xcframework";
```

Reply:
See the note at the end of this section: https://docs.flutter.dev/add-to-app/ios/project-setup#link-on-the-frameworks